server/routes/bitbucket/bitbucket_selection.py

```
@@ -420,6 +437,120 @@ def _store_hook_uuid(hook_uuid: str) -> None:
+    if not need_hooks and not to_enable:
+        return out
+
+    secret = get_or_create_webhook_secret(org_id)
+    webhook_url = f"{_webhook_base_url()}/bitbucket/webhook/{org_id}"
+    out.update({
+        "webhook_url": webhook_url,
+        "webhook_secret": secret,
+        "webhook_events": webhook_events,
+    })
+    if need_hooks:
+        def _one(name):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_one`

(ANN202)